test/com/facebook/buck/python/PythonBinaryIntegrationTest.java

```diff
@@ -248,7 +243,9 @@ public void binaryIsCachedProperly() throws IOException {
 
   @Test
   public void externalPexToolAffectsRuleKey() throws IOException {
-    assumeThat(packageStyle, equalTo(PythonBuckConfig.PackageStyle.STANDALONE));
+    if (packageStyle != PythonBuckConfig.PackageStyle.STANDALONE) {
+      return;
+    }
```

Comment:
This (and below) are mostly cosmetic - it's to get rid of guaranteed assumption violations. In this case the pex tool isn't used for anything but the `STANDALONE` package style - on any platform/installation. It's not an assumption violation - it's just an artifact of parametrized tests that the `externalPexToolAffectsRuleKey[INPLACE,*]` test even exists. It boils down to giving a green build rather than a yellow one.